assets/javascripts/discourse-user-notes/lib/user-notes.js

```diff
@@ -1,22 +1,18 @@
-import showModal from "discourse/lib/show-modal";
+import UserNotesModal from "../../discourse/components/modal/user-notes";
+import { getOwner } from "discourse-common/lib/get-owner";
```

Comment:
I think using this will be triggering a deprecation message in the console? We should update it to use `getOwnerWithFallback`